cli/src/main/scala-2.11/coursier/cli/Helper.scala

```diff
@@ -657,12 +657,12 @@ class Helper(
     }
   }
 
-  def fetch(
+  def fetchMap(
     sources: Boolean,
     javadoc: Boolean,
     artifactTypes: Set[String],
     subset: Set[Dependency] = null
-  ): Seq[File] = {
+  ): Map[String,File] = {
```

Comment:
1. Could you docstring for this interface since the return type is changed?
2. nit: space after `,`

Reply:
Hmm this should cause the CI to fail due to other code using this. Will investigate..

Reply:
Oh I see. `def fetch` still calls this. 

Looks good to me, just nits above. Thanks!